package.json

```diff
@@ -43,6 +43,7 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.29",
     "@fortawesome/free-solid-svg-icons": "^5.13.1",
     "@fortawesome/react-fontawesome": "^0.1.11",
+    "@openmrs/react-root-decorator": "^3.3.1",
```

Comment:
In the new version this was replaced IIRC by the context module(`@openmrs/esm-context`).

Reply:
+1, if you could replace this, that would be great